VideoApp/VideoApp/TwilioVideo/Captions/CaptionsManager.swift

```diff
@@ -22,23 +22,46 @@ import TwilioVideo
 /// is only for receiving captions and should not be visible in the UI.
 class CaptionsManager: NSObject, ObservableObject {
     @Published var captions: [Caption] = []
-    private var subscriptions = Set<AnyCancellable>()
+    @Published var error: Error? = nil
```

Comment:
The logic in this file is a little clunky because of the captions API we are using for private beta. There will be changes for GA so that we don't have to do things like use the presence of a data track to detect to determine error state.